libsolidity/ast/Types.cpp

```diff
@@ -43,6 +43,14 @@ using namespace std;
 using namespace dev;
 using namespace dev::solidity;
 
+namespace
+{
+
+uint32_t static const exponentLimit = 9999;
```

Comment:
We don't really use `uint32_t` explicitly in the rest of the codebase. I think I almost exclusively use `size_t`.

Reply:
`size_t` or `unsigned` is most common in the codebase